tests/test_api.py

```diff
@@ -32,14 +37,19 @@ def test_manifest_csv(helpers):
     test_manifest_path = helpers.get_data_path("mock_manifests/Valid_Test_Manifest.csv")
     yield test_manifest_path
 
+@pytest.fixture(scope="class")
```

Comment:
Note: I added the test manifest using biospecimen component to make sure that it won't mess up with cross validation rules.